src/subscription_manager/repolib.py

```diff
@@ -516,7 +519,12 @@ def get_all_content(self, baseurl, ca_cert):
         release_source = YumReleaseverSource()
 
         # query whether OCSP stapling is advertized by CP for the repositories
-        has_ssl_verify_status = self.get_consumer_auth_cp().has_capability("ssl_verify_status")
+        try:
+            has_ssl_verify_status = self.get_consumer_auth_cp().has_capability("ssl_verify_status")
+        except (ConnectionException, ProxyException) as exc:
+            # Catch connection errors to fix ENT-5215
```

Comment:
I'd make this comment a bit more verbose, e.g. "... to not block the update of the repositories in case of failure to connect to the server: